hatchery/alb.go

```diff
@@ -223,7 +229,10 @@ func (creds *CREDS) terminateLoadBalancerTargetGroup(userName string) error {
 		Region:      aws.String("us-east-1"),
 	})))
 	tgName := truncateString(strings.ReplaceAll(os.Getenv("GEN3_ENDPOINT"), ".", "-")+userToResourceName(userName, "service")+"tg", 32)
-	Config.Logger.Printf("Deleting target group: %s", tgName)
+	Config.Logger.Debug("Deleting target group",
+		"target-group", tgName,
+		"username", userName,
+	)
```

Comment:
won't it be harder to parse these if we use different custom fields everywhere? 🤔  instead of a `target-group` field why not write `Deleting target group XYZ`?

Reply:
That is just extra context, for debugging purposes. I don't think parsing will be harder, no.